<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="TestText" file="D:\Sites\SvetskiPutnik\source\application\tests\models\TestText.php" tests="2" assertions="3" failures="1" errors="0" time="0.381798">
    <testcase name="test_InputValidation" class="TestText" file="D:\Sites\SvetskiPutnik\source\application\tests\models\TestText.php" line="24" assertions="2" time="0.065410">
      <failure type="PHPUnit_Framework_ExpectationFailedException">TestText::test_InputValidation
Failed asserting that '{"error":"Nisu sva polja popunjena","succ":false,"id":"3"}' contains "error":""".

D:\Sites\SvetskiPutnik\source\application\tests\models\TestText.php:31
</failure>
    </testcase>
    <testcase name="test_CreateText" class="TestText" file="D:\Sites\SvetskiPutnik\source\application\tests\models\TestText.php" line="33" assertions="1" time="0.316388"/>
  </testsuite>
</testsuites>
